jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -0,0 +1,218 @@
+// Useful references:
+// - https://pagure.io/fedora-qa/fedora_openqa/blob/main/f/src/fedora_openqa/consumer.py
+// - https://pagure.io/fedora-ci/general/issue/436
+// - https://github.com/json-path/JsonPath
+// - https://sumiya.page/jpath.html
+
+// XXX: we should move this list to a separate YAML file for knob extensibility
+// in the future (for e.g. which tests to run for each)
+def srpms = [
+    'container-selinux',
+    'glibc',
+    'grub2',
+    'ignition',
+    'kernel',
+    'kexec-tools',
+    'NetworkManager',
+    'ostree',
+    'podman',
+    'rpm-ostree',
+    'rust-afterburn',
+    'rust-coreos-installer',
+    'rust-zincati',
+    'selinux-policy',
+    'systemd'
+]
+
+node {
+    // XXX: we should drain what we need of pipeutils into coreos-ci-lib
+    shwrap("rm -rf pipe && git clone https://github.com/coreos/fedora-coreos-pipeline --depth=1 pipe")
+    // these are script global vars
+    pipeutils = load("pipe/utils.groovy")
+    // XXX: should use load_pipecfg but it checks for arch availability
+    pipecfg = readYaml(file: "pipe/config.yaml")
+
+    // query the currently active streams and collect their releasevers; we'll
+    // use these to update our triggers
+    streams = pipeutils.streams_of_type(pipecfg, 'development') +
+              pipeutils.streams_of_type(pipecfg, 'mechanical')
+    stream_to_releasever = [:]
+    for (st in streams) {
+        // not particularly proud of this, but... it'll work for now
+        shwrap("curl -sSLO https://raw.githubusercontent.com/coreos/fedora-coreos-config/${st}/manifest.yaml")
+        def yaml = readYaml(file: "manifest.yaml")
+        stream_to_releasever[st] = yaml.releasever
+    }
+
+    releasevers = stream_to_releasever.values() as Set
+
+    // query which ones use updates-testing; for more context, see
+    // https://github.com/coreos/coreos-ci/pull/49#issuecomment-1769679589
+    composed_releasevers = [] as Set
+    uncomposed_releasevers = [] as Set
```

Comment:
I'm thinking we could come up with better names for composed versus uncomposed, but I'm not really coming up with anything good either.. bodhi_driven versus pungi_driven ?

Reply:
I went with `bodhi_composed_releasevers` and `pungi_composed_releasevers` since I think `releasever` should be in there.